### PR TITLE
[9.0] [MIG] project_task_code module

### PR DESCRIPTION
**Pull request by cubells**
- Updated README
- Updated views
- Updated tests

cc @Tecnativa

### CODE REVIEW

File: project_task_code/README.rst [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
+=========================
 Sequential code for tasks
 =========================
 
 * This module adds a sequential code for tasks.
 
+Usage
+=====
+
+To use this module, you need to:
```
Comment (pedrobaeza) [inline project_task_code/README.rst:23]: Add here that when you create a task in the menu X, you get a code saving it (you know, specifying points with `#.`).

File: project_task_code/README.rst [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
+=========================
 Sequential code for tasks
 =========================
 
 * This module adds a sequential code for tasks.
 
+Usage
```
Comment (pedrobaeza) [inline project_task_code/README.rst:20]: Add configuration section to specify where to change the task code sequence.

File: project_task_code/__openerp__.py [hunk 3]
```diff
@@ -1,35 +1,23 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Sequential Code for Tasks",
-    "version": "8.0.1.0.0",
+    "version": "9.0.1.0.0",
     "category": "Project Management",
     "author": "OdooMRP team, "
               "AvanzOSC, "
               "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
```
Comment (pedrobaeza) [inline project_task_code/__openerp__.py]: You can remove my old company now.

**Review by dreispt — APPROVED** [review]
LGTM!

**Review by pedrobaeza — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
Thanks

**Pull request merged**